.mailmap

```diff
@@ -177,6 +177,8 @@ David M. Karr <davidmichaelkarr@gmail.com>
 David Sheets <dsheets@docker.com> <sheets@alum.mit.edu>
 David Sissitka <me@dsissitka.com>
 David Williamson <david.williamson@docker.com> <davidwilliamson@users.noreply.github.com>
+David Dooling <dooling@gmail.com>
+David Dooling <dooling@gmail.com> <david.dooling@docker.com>
```

Comment:
Can you sort this list? Looks like it's in the wrong position (should be before `David M.` probably)

Reply:
Oh! Looks like the most recent commits from David use his `@docker.com` address in the sign-off, so this should probably be the reverse;

```
David Dooling <david.dooling@docker.com>
David Dooling <david.dooling@docker.com> <dooling@gmail.com>
```

Reply:
I assumed here that @whalelines preferred the original email here; can you confirm?